src/Microsoft.Health.Fhir.Stu3.Web/appsettings.json

```diff
@@ -22,7 +22,7 @@
             "SupportsXml": true
         },
         "CoreFeatures": {
-            "SupportsBatch": false,
+            "SupportsBatch": true,
```

Comment:
Just curious, is this something that's ready for prime time and we want to enable by default now? If it's for testing only, there is a test configuration override that you can do (you can reference corstestconfiguration.json for example).

Reply:
@brandonpollett Thoughts?

Reply:
The original thought was to enable it by default now, but I could see the use of holding off until we get a bit more testing complete and the transaction tests in, too. 

@rbans96 - You can look into how this functions with the `corstestconfiguration.json` or `exporttestconfiguration.json` which are located in the `test\Configuration` folder. It will also require a bit of a change in the build system to make work which I can help with.